src/OrbitGl/TimeGraph.cpp

```diff
@@ -238,13 +229,12 @@ void TimeGraph::VerticallyMoveIntoView(const TimerInfo& timer_info) {
 void TimeGraph::VerticallyMoveIntoView(Track& track) {
   float pos = track.GetPos()[1];
   float height = track.GetHeight();
-  float world_top_left_y = viewport_->GetScreenTopLeftInWorld()[1];
 
-  float max_world_top_left_y = pos;
-  float min_world_top_left_y =
-      pos + height - viewport_->GetVisibleWorldHeight() + layout_.GetBottomMargin();
-  viewport_->SetScreenTopLeftInWorldY(
-      std::clamp(world_top_left_y, min_world_top_left_y, max_world_top_left_y));
+  float min_vertical_scrolling_offset = pos;
+  float max_vertical_scrolling_offset =
+      pos + height - viewport_->GetWorldHeight() + layout_.GetBottomMargin();
+  SetVerticalScrollingOffset(std::clamp(vertical_scrolling_offset_, max_vertical_scrolling_offset,
+                                        min_vertical_scrolling_offset));
```

Comment:
Given `clamp( const T& v, const T& lo, const T& hi ); `Are max and min in the correct order here? If so, maybe a comment could be helpful.